pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -0,0 +1,1031 @@
+
+  // Instrument queries
+  private static final String queryAllInstruments = "SELECT i.instrumentId, i.name, i.platformId " + "FROM Instrument AS i";
+  private static final String queryInstrumentById = queryAllInstruments + " WHERE i.instrumentId = ?";
+  private static final String queryInstrumentsByModelId = queryAllInstruments + " WHERE i.platformId = ?";
+
+  // Order queries
+  private static final String queryAllOrders = getResourceAsString("queryAllOrders.sql");
+  private static final String queryOrderById = queryAllOrders + " WHERE poolOrderId = ?";
+  private static final String queryAllOrderSamples = getResourceAsString("queryAllOrderSamples.sql");
+  private static final String queryOrderSamplesByOrderId = queryAllOrderSamples + " WHERE poolOrderId = ?";
+
+  // User queries
+  private static final String queryAllUsers = "SELECT u.userId, u.fullname, u.email, u.active " + "FROM User AS u";
+  private static final String queryUserById = queryAllUsers + " WHERE u.userId = ?";
+
+  // Run queries
+  private static final String queryAllRuns = getResourceAsString("queryAllRuns.sql");
+  private static final String queryRunById = queryAllRuns + " AND r.runId = ?";
+  private static final String queryRunByName = queryAllRuns + " AND r.alias = ?";
+
+  // RunPosition queries
+  private static final String queryAllRunPositions = getResourceAsString("queryAllRunPositions.sql");
+  private static final String queryRunPositionsByRunId = queryAllRunPositions + " WHERE r_spc.Run_runId = ?";
+
+  // RunSample queries
+  private static final String queryAllRunSamples = getResourceAsString("queryAllRunSamples.sql");
+  private static final String queryRunSamplesByRunId = queryAllRunSamples
+      + " JOIN SequencerPartitionContainer_Partition spcp ON spcp.partitions_partitionId = part.partitionId"
+      + " JOIN SequencerPartitionContainer spc ON spc.containerId = spcp.container_containerId"
+      + " JOIN Run_SequencerPartitionContainer rcpc ON rcpc.containers_containerId = spc.containerId" + " WHERE rcpc.Run_runId = ?";
+
+  // Sample queries
+  private static final String queryAllSamples = getResourceAsString("queryAllSamples.sql");
+  private static final String querySampleById = "SELECT * FROM (" + queryAllSamples + ") combined " + "WHERE id = ?";
+
+  private static final String querySampleChildIdsBySampleId = getResourceAsString("querySampleChildIdsBySampleId.sql");
+
+  // SampleType (MISO SampleClass and Library) queries
+  private static final String queryAllSampleTypes = getResourceAsString("queryAllSampleTypes.sql");
+
+  // SampleProject queries
+  private static final String queryAllSampleProjects = getResourceAsString("queryAllSampleProjects.sql");
+
+  // SampleChangeLog queries
+  private static final String queryAllSampleChangeLogs = getResourceAsString("queryAllSampleChangeLogs.sql");
+  private static final String querySampleChangeLogById = "SELECT * FROM (" + queryAllSampleChangeLogs + ") combined "
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)